### ISSUE TITLE
Comment notifications are not marked as read when replied to

### ISSUE THREAD

**Issue by Tmpod** [LemmyNet/lemmy-ui#442]
### Issue Summary

When a comment is replied to from the notification page, it is not marked as read, and instead has to be manually checked.

### Steps to Reproduce

1. Go to your notifications page and unmark an old notif if you don't have new ones.
2. Reply to it

### Technical details
N/A

### GIF

![Peek 2021-10-02 13-03](https://user-images.githubusercontent.com/35771816/135715307-4d542e7c-b094-4c5a-acee-7171df5eae98.gif)

**Comment by dessalines** [comment]
Hrm yeah that does make sense. I'm wondering if this should be included in the back end instead.... but doing this on the front end for now should be fine.

**Comment by Tmpod** [comment]
Yeah, it could be done on the back-end, but doing it here would probably be faster. Although, if it isn't to much effort (I don't believe it is, but I don't know the rust codebase), I'd say it would be better to just implement it on the back-end right away, instead of postponing it (and likely forget it).

**Comment by dessalines** [comment]
@Nutomic thoughts on this? Maybe the back-end is the way to go for this one. I can't think of any situation where you reply to a comment, and NOT want the comment you're replying to to be marked as read.

**Comment by Nutomic** [comment]
I'm pretty undecided on this. Now "mark as read" is effectively handled by the frontend when you press a button. So it might be surprising when it is done automatically by the backend. But doing it in the backend would mean less work and less api calls for frontends/clients. So either one works for me.

**Comment by Tmpod** [comment]
Yeah, good point. Maybe we could ask around in the Matrix channel to see what other people think of this?

**Comment by LunaticHacker** [comment]
I think doing it in the backend makes more sense. either way plz assign me when we reach a consensus :)

**Comment by Tmpod** [comment]
hehe, nice. I think we should move forward with the back-end then ;)
